code/modules/clothing/suits/power_armor.dm

```diff
@@ -167,8 +167,8 @@
 	requires_training = TRUE
 	powered = TRUE
 	slowdown = 0.24
-	armor_block_chance = 60
-	deflection_chance = 10 //20% chance to block damage from blockable bullets and redirect the bullet at a random angle
+	armor_block_chance = 30
+	deflection_chance = 0 //20% chance to block damage from blockable bullets and redirect the bullet at a random angle
```

Comment:
You can just remove all of the liens you set to 0 instead of leaving them in.

Reply:
Frok wanted me to set the values to 0. That’s why i did that.